web/ui/static/js/graph/index.js

```diff
@@ -513,6 +513,10 @@ Prometheus.Graph.prototype.submitQuery = function() {
           return;
         }
 
+        if ("warnings" in json && json.warnings.length > 0) {
+          self.showWarning(json.warnings.join('<br>'));
```

Comment:
We should probably prefix this with an indication that it's a remote read issue.

Reply:
May be it wpuld be better to add it within error from storage/remote like fmt.Sprintf("remote_read: %s", err). How do you think?

Reply:
Makes sense.

Reply:
Prefix was added